packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -47,6 +47,13 @@ abstract class ScrollActivityDelegate {
   /// Terminate the current activity and start a ballistic activity with the
   /// given velocity.
   void goBallistic(double velocity);
+
+  /// Update the ballistic animation instead of restarting  it, for example as
+  /// the result of a layout change after a flinging gesture.
+  ///
+  /// The [initVelocity] and [initPosition] refer to the starting values of the
+  /// new ballistic animation.
+  Simulation? updateBallisticAnimation(double initVelocity, double initPosition);
```

Comment:
From original PR: Add docs for when does it return null and why/what does it mean?

Source: https://github.com/flutter/flutter/pull/100133/files#r844474467

Reply:
Added.